build.gradle

```diff
@@ -1,10 +1,17 @@
+import com.github.jk1.license.filter.LicenseBundleNormalizer
+import com.github.jk1.license.render.JsonReportRenderer
+
 plugins {
     id 'java'
     id 'java-library'
     id 'maven-publish'
-    id "io.github.gradle-nexus.publish-plugin" version "1.1.0"
+    id "io.github.gradle-nexus.publish-plugin" version "1.3.0"
+    id "org.owasp.dependencycheck" version "9.0.9"
```

Comment:
I am not sure if you have seen this [release note](https://github.com/jeremylong/DependencyCheck#900-upgrade-notice) but I believe we must start migrating to NVD API and get the API key for our company